test/e2e/utils/common.js

```diff
@@ -6,10 +6,485 @@ const timeouts = {
+    field: 'input[name="isCoveredByHealthInsurance-0"] + label'
+  },
+
+  providers: [
+    {
+      insuranceName: {
+        field: 'input[name="insuranceName"]',
+        value: 'BCBS'
+      },
+      insurancePolicyHolderName: {
+        field: 'input[name="insurancePolicyHolderName"]',
+        value: 'William Shakespeare'
+      },
+      insurancePolicyNumber: {
+        field: 'input[name="insurancePolicyNumber"]',
+        value: '100'
+      },
+      insuranceGroupCode: {
+        field: 'input[name="insuranceGroupCode"]',
+        value: '200'
+      },
+    }
+  ],
+
+  isMedicaidEligible: {
+    field: 'input[name="isMedicaidEligible-1"] + label'
+  },
+  isEnrolledMedicarePartA: {
+    field: 'input[name="isEnrolledMedicarePartA-1"] + label'
+  },
+  medicarePartAEffectiveDate: {
+    month: {
+      field: 'select[name="medicarePartAEffectiveMonth"]',
+      value: 'Apr'
+    },
+    day: {
+      field: 'select[name="medicarePartAEffectiveDay"]',
+      value: '23'
+    },
+    year: {
+      field: 'input[name="medicarePartAEffectiveYear"]',
+      value: '1980'
+    }
+  },
+
+  lastServiceBranch: {
+    field: 'select[name="lastServiceBranch"]',
+    value: 'army'
+  },
+  lastEntryDate: {
+    month: {
+      field: 'select[name="lastEntryMonth"]',
+      value: 'Oct'
+    },
+    day: {
+      field: 'select[name="lastEntryDay"]',
+      value: '10'
+    },
+    year: {
+      field: 'input[name="lastEntryYear"]',
+      value: '2000'
+    }
+  },
+  lastDischargeDate: {
+    month: {
+      field: 'select[name="lastDischargeMonth"]',
+      value: 'Nov'
+    },
+    day: {
+      field: 'select[name="lastDischargeDay"]',
+      value: '11'
+    },
+    year: {
+      field: 'input[name="lastDischargeYear"]',
+      value: '2004'
+    }
+  },
+  dischargeType: {
+    field: 'select[name="dischargeType"]',
+    value: 'honorable'
+  },
+
+  purpleHeartRecipient: {
+    field: 'input[name="purpleHeartRecipient"] + label'
+  },
+  isFormerPow: {
+    field: 'input[name="isFormerPow"] + label'
+  },
+  postNov111998Combat: {
+    field: 'input[name="postNov111998Combat"] + label'
+  },
+  disabledInLineOfDuty: {
+    field: 'input[name="disabledInLineOfDuty"] + label'
+  },
+  swAsiaCombat: {
+    field: 'input[name="swAsiaCombat"] + label'
+  },
+  vietnamService: {
+    field: 'input[name="vietnamService"] + label'
+  },
+  exposedToRadiation: {
+    field: 'input[name="exposedToRadiation"] + label'
+  },
+  radiumTreatments: {
+    field: 'input[name="radiumTreatments"] + label'
+  },
+  campLejeune: {
+    field: 'input[name="campLejeune"] + label'
+  }
+};
+
+function completePersonalInformation(client, data, onlyRequiredFields) {
   client
-    .setValue('input[name="fname"]', 'William')
-    .setValue('input[name="lname"]', 'Shakespeare');
+    .setValue(data.veteranFullName.first.field, data.veteranFullName.first.value)
+    .setValue(data.veteranFullName.last.field, data.veteranFullName.last.value);
```

Comment:
I think that defining the `field` attribute may be half a step too far - I'd expect those to be consistent between the tests, no matter the `value`.